src/katgpucbf/fgpu/compute.py

```diff
@@ -212,8 +212,6 @@ def __init__(
             "gains": ["postproc:gains"],
         }
         aliases = {}
-        if template.ddc is not None:
-            compounds["ddc_weights"] = [f"ddc{pol}:weights" for pol in range(N_POLS)]
```

Comment:
Not directly related to this PR; this is no longer useful because the DDC weights are now managed internally by the DDC class and are not slots.